project/summary/api.py

```diff
@@ -1,8 +1,25 @@
-from assessment.api import AssessmentViewset, DisabledPagination
+from rest_framework.filters import BaseFilterBackend
+from assessment.api import AssessmentViewset, DisabledPagination, InAssessmentFilter
 
 from . import models, serializers
 
 
+class UnpublishedFilter(BaseFilterBackend):
+    """
+        Only show unpublished visuals to admin and assessment members.
+    """
+    def get_unpublished_perms(self, user, view):
```

Comment:
this fails if a user is not a superuser and they're viewing a retrieve view; view.assessment doesn't exist.